CHANGELOG.md

```diff
@@ -0,0 +1,16 @@
+# Change Log
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/)
+and this project adheres to [Semantic Versioning](http://semver.org/).
+
+## [Unreleased]
+
+## [0.0.2] - 2017-03-30
+### Added
+- a `CHANGELOG.md` 🎉
+
+### Changed
+- Quick Start example was not able to be copypasta'd, and had props
+mixed up between components.
+- removed lodash from the bundle by only importing intersection
```

Comment:
maybe "removed full lodash" to be more explicit? lodash is still in da bundle, just a smaller portion of it